README.md

```diff
@@ -107,6 +107,23 @@ Docker to work with the helper.
 To build and install the Amazon ECR Docker Credential Helper, we suggest Go
 1.9+, `git` and `make` installed on your system.
 
+If you just installed Go, make sure you also have added it to your PATH or 
+Environment Vars (Windows). For example:
+
+```
+$ export GOPATH=$HOME/gocode
```

Comment:
nit: In recent versions of Go (1.10 or greater), the GOPATH is defaulted to `~/go`.  I'd prefer that we recommend `$HOME/go` or `~/go` to stay consistent.

Reply:
Sounds good!

Reply:
Fixed.